rudder-agent/SOURCES/rudder.init

```diff
@@ -0,0 +1,219 @@
+#!/bin/sh
+
+#==============================================
+# Start/stop script for all Rudder components #
+#==============================================
+#
+### BEGIN INIT INFO
+# Provides:		rudder
+# Required-Start:	$local_fs $remote_fs $network $syslog
+# Required-Stop:	$local_fs $remote_fs $network $syslog
+# Default-Start:	2 3 4 5
+# Default-Stop:		0 1 6
+# Short-Description:	Rudder server
+# Description:		Rudder server including all components required to run
+### END INIT INFO
+#
+# Copyright (C) 2011-2015 Normation
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# GPL License: http://www.gnu.org/licenses/gpl.txt
+#
+#====================================================================
+
+# Variables
+
+## Colors configuration
+ECHO_BIN="/bin/echo -e"
```

Comment:
(Just as a note: We do not use this script on AIX, SRC takes care of the init stuff instead :) )